www/index.html

```diff
@@ -84,12 +84,12 @@ <h5 class="card-subtitle mb-2 text-muted">Short but descriptive sub-heading</h5>
               content. This content is a little bit longer.</p>
 
             <p id="total-transactions" class="card-text">
-              55.000 unconfirmed Transactions (37 MB)</p>
+              0 unconfirmed Transactions (0 MB)</p>
 
             <label for="input-txid-lookup">Transaction Lookup</label>
             <div class="input-group mb-3">
               <input type="text" id="input-lookup-txid" class="form-control"
-                placeholder="4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b" aria-label="txid"
+                placeholder="Your transaction ID e.g. 4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b" aria-label="txid"
```

Comment:
Would just keep the txid hash. Placeholders show the specify format a user input should have.